test/src/main/configurations/UTIL/Larva/xsl/CreateMail.xsl

```diff
@@ -16,11 +18,18 @@
 		</xsl:choose>
 	</xsl:variable>
 
+	<xsl:variable name="ccList">
+		<xsl:choose>
+			<xsl:when test="contains($message, 'All scenarios passed')"><xsl:value-of select="$cc"/></xsl:when>
```

Comment:
Ricardo,
You now determine if there are failed tests by parsing the test results.
I would prefer that you configure a forward named 'fail' on the LarvaPipe. That forward is used when there are failed tests.